src/guacd/conf-file.h

```diff
@@ -32,10 +37,17 @@
 int guacd_conf_parse_file(guacd_config* conf, int fd);
```

Comment:
What about hiding `guacd_conf_parse_file()` by making it static?
Now, `guacd_conf_load` provides this functionality on higher level.

Reply:
My only comment here would be to stay consistent with how this is done elsewhere in the code.  Are other functions hidden using static, or just not stubbed in header files?  Go with the route that provides the most consistency, unless there's a really good reason to do otherwise.